config/application.rb

```diff
@@ -37,4 +37,10 @@ class Application < Rails::Application
 
     config.assets.paths << Rails.root.join("app", "assets", "fonts")
   end
+
+  def self.version
+    if File.exist?(Rails.root.join("VERSION"))
```

Comment:
We usually have an `else` statement defaulting to `development` (or `development-#{commit-short-hash}` if we know `git` is available).

See [`instedd/guisso`](https://github.com/instedd/guisso/blob/7015ba06216772214981a63af3c887ffb30fc3d2/config/initializers/version.rb) for an example.